src/index.ts

```diff
@@ -44,14 +58,14 @@ class GraphqlJsonToSdl extends Command {
 
 class EmptySchemaError extends Error { }
 
-function writeSchema(src: string, out: string) {
+function writeSchema(src: string, out: string, raw?: boolean) {
   const fileContent = fs.readFileSync(src, "utf-8");
 
   if (!fileContent) {
     throw new EmptySchemaError();
   }
 
-  const { data } = JSON.parse(fileContent);
+  const { data } = raw ? { data: JSON.parse(fileContent) } : JSON.parse(fileContent);
```

Comment:
Rather than using a flag, it should be OK to parse the file and then check if the `data` property exists. If the property exists, use that. If not, use the parsed contents directly.